R_Exercises8.Rmd

```diff
@@ -23,25 +23,58 @@ Check your work by knitting the document.
 
 1. Create a level 2 header below in Markdown with the name "Answer 1".
 
+## "Answer 1"
 
 2. Add an unordered Markdown list of your favorite food dishes and
 add their ingredients as sub-items in the list. You should enter at least two
 dishes with two ingredients each.
 
-
+* Quinoa poha
+  * Quinoa(boiled)
+  * Curry leaves
+  * Onion
+  * Potato
+  * Peanuts
+   
+   
+* Red Thai curry(vegan)
+  * Red Thai paste
+  * Coconut milk
+  * Vegetables
+  * Tofu
+  
+   
+* PaniPuri
+  * Raw mango
+  * Mint
+  * Coriander
+  * Green chillies
+  * Rock salt
+  * Cumin powder
+  * Water
 3. Insert an R code chunk below. Show the `summary` of the `Orange` dataset.
 Run a linear regression using the `lm` function to predict `circumference` from
 `age` and assign the result. Show the `summary` of the fit object that you
 assigned.
 
+```{r}
+
+summary(Orange)
+
+eagle <- lm(circumference~age, data = Orange)
 
+summary(eagle)
+```
 4. Add an RMarkdown weblink (in link format) below to your favorite R reference
 website.
 
+https://www.rstudio.com/resources/cheatsheets/
```

Comment:
The link should look like this \[ website \]\( URL \)

Reply:
> The link should look like this [ website ]( URL )
I see, got it. Thank you professor.